examples/todoapp/test/feature_todo_list/blocs/todo_list_test.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+import 'package:rx_bloc/rx_bloc.dart';
+import 'package:rx_bloc_test/rx_bloc_test.dart';
+import 'package:todoapp/base/common_blocs/coordinator_bloc.dart';
+import 'package:todoapp/base/common_services/todo_list_service.dart';
+import 'package:todoapp/base/models/todo_model.dart';
+import 'package:todoapp/base/models/todos_filter_model.dart';
+import 'package:todoapp/feature_todo_list/blocs/todo_list_bloc.dart';
+
+import '../../Stubs.dart';
+import 'todo_list_test.mocks.dart';
+
+@GenerateMocks([
+  TodoListService,
+  CoordinatorBlocType,
+  CoordinatorEvents,
+  CoordinatorStates,
+])
+void main() {
+  late TodoListService _todoListService;
+  late CoordinatorBlocType _coordinatorBloc;
+  late CoordinatorEvents _coordinatorBlocEvents;
+  late CoordinatorStates _coordinatorBlocStates;
+
+  void _defineWhen() {
```

Comment:
As stated above it will be great to have the flexibility if changing the data through method arguments

Reply:
@nikolay-vasilev-prime do we have any updates here ?

Reply:
updated it